gluten-core/src/main/scala/io/glutenproject/expression/ExpressionMappings.scala

```diff
@@ -143,6 +143,8 @@ object ExpressionMappings {
   // JSON functions
   final val GET_JSON_OBJECT = "get_json_object"
   final val JSON_ARRAY_LENGTH = "json_array_length"
+  final val TO_JSON = "to_json"
```

Comment:
@lviiii please check whether velox backend supports these function.

Reply:
Velox backend don't support these function and Gluten will take the fallback strategy automatically.